AnnoDesigner/Undo/IUndoableOperation.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace AnnoDesigner.Undo
+{
+    public interface IUndoableOperation
```

Comment:
My opinion is to rename it to `IOperation` and move it to a namespace like `AnnoDesigner.Undo.Operations`.